glean_parser/__main__.py

```diff
@@ -206,6 +224,7 @@ def main(args=None):
 main.add_command(check)
 main.add_command(glinter)
 main.add_command(coverage)
+main.add_command(data_review_request, "data_review_request")
```

Comment:
this now gets invoked as `glean_parser data_review_request ...`.
Underscores in commands somehow feel icky to me (plus it's long to type).

Maybe just `data-review`?